examples/image_classification-tf.ipynb

```diff
@@ -0,0 +1,7883 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    push_to_hub_model_id = f"{model_name}-finetuned-eurosat"
</pre><p> </p><p>Is it missing the <code>username</code> variable here? Or <code>notebook_login()</code> will automatically take care of this?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p><code>notebook_login()</code> takes care of it, but you can also specify it e.g. <code>hub_model_id='username/model-checkpoint'</code> . You'll need to have write permission to push to the specified repo name/username</p>